code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -189,7 +189,7 @@
 		heretic_datum.high_value_sacrifices++
 
 	heretic_datum.total_sacrifices++
-	heretic_datum.knowledge_points += 2
+	heretic_datum.knowledge_points += 4 //SKYRAT EDIT - ORIGINAL: 2
```

Comment:
sacrifices didn't need a point buff, espiecally with how strong rifts are now.

Reply:
Rifts aren't any stronger unless you spend points to make the book to buff them, which is a risky investment.
I'm willing to lower it down to three points, but I still think they should get a point buff given that you aren't allowed to JUMP people here, in the same way you would be on TG normally.

Reply:
After thinking about it, I have decided to make it only one more point for the moment.
I'll see how it feels after I watch a couple shifts play out with the minor buff.